internals/secrethub/keyring.go

```diff
@@ -44,13 +44,29 @@ type PassphraseReader interface {
 
 // passphraseReader provides passphrase reading capability to the CLI.
 type passphraseReader struct {
+	tries     int
+	hasAsked  bool
 	io        ui.IO
 	FlagValue string
 	Cache     *PassphraseCache
 }
 
+func (pr *passphraseReader) Read() ([]byte, error) {
+	defer func() { pr.tries++ }()
+
+	if pr.tries > 0 {
+		_ = pr.Cache.Delete()
```

Comment:
How do we clear the last try from the cache?

Reply:
Will be cleared the next time the passphrase is read. Not perfect, but it works.